.deploystack/test

```diff
@@ -112,20 +141,33 @@ section_open "Testing Online Boutique's services are running"
     evalTest "kubectl get deployment shippingservice --no-headers -o=name 2> /dev/null"  "deployment.apps/shippingservice"
 section_close
 
-sleep 60
+sleep 120
 
 ENDPOINT=$( kubectl get service frontend-external --no-headers 2> /dev/null | awk '{print $4}')
 
 section_open "Testing Online Boutique's front-end is working"
     evalTest 'curl -s -o /dev/null -w "%{http_code}" $ENDPOINT' "200"
 section_close
 
-terraform -chdir="$DIR" destroy --auto-approve -var gcp_project_id="${PROJECT}" -var name="${NAME}" -var region="${REGION}" -var namespace="${NAMESPACE}"  -var filepath_manifest="${FILEPATH_MANIFEST}" -var memorystore="${MEMORYSTORE}"
+terraform -chdir="$DIR" destroy -auto-approve \
+    -var gcp_project_id="${PROJECT}" \
+    -var name="${NAME}" \
+    -var region="${REGION}" \
+    -var namespace="${NAMESPACE}"  \
+    -var filepath_manifest="${FILEPATH_MANIFEST}" \
+    -var memorystore="${MEMORYSTORE}"
 
 section_open "Testing Google Kubernetes Engine cluster does NOT exist"
     evalTest 'gcloud container clusters describe online-boutique --format="value(name)" --region $REGION' "EXPECTERROR"
 section_close
 
+# This is only needed if you tests fail alot because of overlaping runs of the
+# same set of tests. Really don't do this if you don't want to severly irritate 
+# @tpryan
+section_open "Delete Test Project"
+    gcloud projects delete $PROJECT -q
+section_close
```

Comment:
**Thought:**
Previously, we had a `clean` bash script that cleaned up GCP resources.
The `clean` bash script ran as a separate step inside the Google Cloud Build Trigger (`.deploystack/test.yaml`).
Also, `clean` ran _before_ `test`, so I'm guessing failures in `test` didn't step `clean` from running.
It it possible that we end up with if failures occur in the `test` script before we reach `gcloud projects delete $PROJECT -q`?
```
gcloud container clusters delete online-boutique --region $REGION -q
```

Just a thought. No action needed.

Reply:
Yes that's actually a feature, not a bug.  If a test fails the project remains for forensics.

The DeployStack platform will prune these projects after some predetermined length of time. I need to write up the procedure and document how to get access to the projects and pause deletion. .

Reply:
> "If a test fails the project remains for forensics."

That totally makes sense!